### DISCUSSION TITLE
Test realtime Concurrent connection limit

### DISCUSSION THREAD

**Discussion by ioRekz** [supabase/supabase#18081] [Questions]
Hello, I'm building an app for Twitch streamers that could get above the concurrent connections limit. 

I would like to test how my application behaves when it reaches this limit. Is there a way to mock 200 connections (I'm on the free plan si far) or temporarily reduce the limit to a low number to manually test?

**Comment by GaryAustin1** [comment]
I did this awhile back.   Maybe it or something like it still works.
```
    for (let i=0; i< 250; i++) {
        const client = new supabase.RealtimeClient('wss://xxxxxxxxxxxxxxxxxxxxx.supabase.co/realtime/v1', {
            params: {
                apikey: SUPABASE_KEY,
                eventsPerSecond: 10,
            },
        })
 
        const channel = client.channel('db-changes')
        channel.on('postgres_changes', {event: '*', schema: 'public'}, (payload) => {
            console.log('All changes in public schema: ', payload)
        }).subscribe()
    }
    ```
